packages/massa-proto-cli/package.json

```diff
@@ -12,14 +12,14 @@
   "dependencies": {
     "@massalabs/as-types": "^1.0.0",
     "@massalabs/massa-as-sdk": "^2.1.0",
-    "@massalabs/massa-web3": "^1.19.0",
+    "@massalabs/massa-web3": "^1.19.1-dev.20230622134858",
```

Comment:
can you create an issue to update the package.json once the new version of massa-web3 is out ?

Reply:
#232